Gemfile

```
@@ -75,9 +75,7 @@ group :test do
   gem 'webmock'
 end
 
-# pin arclight to v1.0.* until we address some changes in v1.1.0
-# documented here: https://github.com/sul-dlss/stanford-arclight/issues/404
-gem 'arclight', '~> 1.0.0'
+gem 'arclight', '>= 1.1.0'
```

I think we should be pessimistic and add a < 2 here:
```suggestion
gem 'arclight', '>= 1.1.0', '< 2'
```

---

:) I had that and deleted it! Will do.